package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "privacy-grade",
-  "version": "1.0.0",
-  "description": "DuckDuckGo's privacy grade calculator",
+  "version": "0.9.0",
```

Comment:
Called this version 0.9.0 to keep things kind of SEMVER-consistent. 1.0.0 would imply a production-ready module.